### PR TITLE
SoundWire: log cleanups

### PR DESCRIPTION
**Pull request by bardliao**
cosmetic cleanups to remove warning for dapm widgets and make enumeration logs more consistent.

### CODE REVIEW

File: drivers/soundwire/bus.c [hunk 1]
```diff
@@ -696,9 +696,9 @@ void sdw_extract_slave_id(struct sdw_bus *bus,
 	id->class_id = SDW_CLASS_ID(addr);
 
 	dev_dbg(bus->dev,
-		"SDW Slave class_id %x, part_id %x, mfg_id %x, unique_id %x, version %x\n",
-				id->class_id, id->part_id, id->mfg_id,
-				id->unique_id, id->sdw_version);
+		"SDW Slave class_id 0x%02x, part_id 0x%04x, mfg_id 0x%04x, unique_id 0x%x, version 0x%x\n",
+		id->class_id, id->part_id, id->mfg_id,
+		id->unique_id, id->sdw_version);
```
Comment (RanderWang) [inline drivers/soundwire/bus.c]: merge  these two line into one ?   > 100 ?

File: drivers/soundwire/slave.c [hunk 2]
```diff
@@ -163,13 +163,13 @@ int sdw_acpi_find_slaves(struct sdw_bus *bus)
...
+					"Valid unique IDs 0x%x 0x%x for Slave mfg_id 0x%04x, part_id 0x%04x\n",
 					id.unique_id, id2.unique_id,
 					id.mfg_id, id.part_id);
 				ignore_unique_id = false;
 			} else {
 				dev_err(bus->dev,
-					"Invalid unique IDs %x %x for Slave mfg %x part %d\n",
+					"Invalid unique IDs 0x%x 0x%x for Slave mfg_id 0x%04x, part_id 0x%04x\n",
 					id.unique_id, id2.unique_id,
 					id.mfg_id, id.part_id);
```
Comment (RanderWang) [inline drivers/soundwire/slave.c]: also here

File: drivers/soundwire/bus.c (hunk 1 above)
Comment (bardliao) [reply drivers/soundwire/bus.c]: > merge these two line into one ? > 100 ?

Fixed. Thanks @RanderWang

**Review by RanderWang — APPROVED** [review]
LGTM

**Review by shumingfan — APPROVED** [review]

**Pull request merged**